api-server/src/server/boot/randomAPIs.js

```diff
@@ -14,36 +14,10 @@ module.exports = function (app) {
   router.get('/u/:email', unsubscribeDeprecated);
   router.get('/unsubscribe/:email', unsubscribeDeprecated);
   router.get('/ue/:unsubscribeId', unsubscribeById);
-  router.get(
-    '/the-fastest-web-page-on-the-internet',
```

Comment:
<img width="461" alt="image" src="https://user-images.githubusercontent.com/63889819/153646489-c259552e-171e-402e-a649-28c0cde51948.png">

Is this handled elsewhere then?

Reply:
It is, yes. It's handled by Gatsby these days: https://github.com/freeCodeCamp/freeCodeCamp/tree/main/client/static/the-fastest-web-page-on-the-internet